src/client/utils/dom.js

```diff
@@ -347,7 +347,7 @@ export function isHammerheadAttr (attr) {
 }
 
 export function isIframeElement (el) {
-    return getTagName(el) === 'iframe';
+    return getTagName(el) === 'iframe' && isDomElement(el);
```

Comment:
Additional check `domUtils.isDomElement` needs only in code instrumentation.
If we will add it to the `domUtils.isSomeElement` then we get proxy perfomance degradation.
@churkin Are you sure that we should do it?